sapp/ui/issues.py

```diff
@@ -462,26 +476,100 @@ def where_filter(self, filter_instance: Filter) -> "Instance":
             .where_status_is_any_of(filter_instance.statuses)
         )
 
+        if filter_instance.callables is not None and len(filter_instance.callables) > 0:
+            if type(filter_instance.callables) is not list:
+                if filter_instance.callables.get("operation") == "matches":
+                    builder = builder.where_callables_matches(
+                        filter_instance.callables.get("value", [""])[0]
+                    )
+                elif len(filter_instance.callables.get("value", [])) > 0:
+                    builder = builder.where_callables_is_any_of(
+                        filter_instance.callables.get("value", ["%"])
```

Comment:
We have a default value of `["%"]` here, but nowhere else. I get that you're propagating the default value of `%` that you deleted from the `Query` object below, but I think that default was needed due to the unconditional `where_callables_is_any_of` that was being done above. Now that you've removed that, we'll only run the `where_callables_is_any_of` check if we actually have something to filter on, so I don't think the default is needed any more.

Reply:
You are right! I was being too defensive (code) there!